modules/tasks/tasks/src/main/java/org/motechproject/tasks/service/impl/ChannelServiceImpl.java

```diff
@@ -177,6 +181,49 @@ protected void doInTransactionWithoutResult(TransactionStatus status) {
         return channels;
     }
 
+    private void registerLoggerChannel() {
```

Comment:
Also, why is this even in the Tasks module? Probable belongs better to where the commons data provider is (that is server-bundle)